### PR TITLE
Added the ERD Gem to the default setup

### PR DESCRIPTION
**Pull request by prawl**
## Why?
- @dacur and I found that when on-boarding devs onto projects, generating the ERD (class architecture) is really helpful for seeing a current snapshot of what our models and relationships looks like.  
- https://github.com/smashingboxes/boxcar/issues/82

## What Changed?
- Added the ERD gem by default to our rails projects
- ERD Gem: https://github.com/voormedia/rails-erd
- Refactored the specs to check that all gems are present

### CODE REVIEW

File: templates/.erdconfig [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+filetype: pdf
+indirect: true
+inheritance: false
+markup: true
+notation: bachman
+orientation: vertical
+polymorphism: false
+sort: true
+warn: true
+title: Kabook - Data Model
```
Comment (dacur) [inline templates/.erdconfig]: i'd just make this the generic "Data Model" or something similar

**Review by dacur — APPROVED** [review]
just need to remove the reference to `Kabook` 👍 

thanks for adding this 🙏

File: spec/support/shared_examples/basic_setup_steps.rb [hunk 2]
```diff
@@ -31,13 +45,19 @@
...
+
   it "gitignores secrets.yml" do
     gitignore = IO.read("#{project_path}/.gitignore")
     expect(gitignore).to match(%r{^/config/secrets.yml$})
   end
 
-  it "adds the annotate gem" do
-    expect(gemfile).to match(/gem "annotate"/)
+  it "includes all of the defaults gems" do
+    DEFAULT_GEMS.each do |gem|
```
Comment (dkniffin) [inline spec/support/shared_examples/basic_setup_steps.rb:58]: 👏

File: templates/.erdconfig (hunk 1 above)
Comment (dkniffin) [reply templates/.erdconfig]: Another option if you do want the project name in there is to make this a `.erb` template file, and fill it in with `<%= app_name.humanize %>`

**Review by dkniffin — APPROVED** [review]

**Comment by dkniffin** [comment]
@prawl Just a reminder that this is still open

**Comment by dacur** [comment]
change looks good, nice work @prawl 👍 👍 👍

**Comment by prawl** [comment]
🏆

**Pull request merged**